test/groups/group_utils.jl

```diff
@@ -271,9 +268,9 @@ function test_group(
     end
 
     test_invariance && @testset "metric invariance" begin
-        if has_invariant_metric(G, LeftAction()) === Val(true)
+        if invariant_metric_dispatch(G, LeftAction()) === Val(true)
```

Comment:
I think you can still use `has_invariant_metric` here?

Reply:
Yes, of course.  Actually seeing these lines in code was one of the reasons I thought about having the `_distpatch` functions that one redefines if necessary and having `is_`, `has_` functions (that internally check the dispatch functions) for ease of use in programs.